src/main/java/nextstep/subway/applicaion/LineService.java

```diff
@@ -26,4 +44,18 @@ public LineResponse saveLine(LineRequest request) {
                 line.getModifiedDate()
         );
     }
+
+    public LineResponse findLineById(Long id) {
```

Comment:
```suggestion
    public LineResponse findBy(Long id) {
```

1. LineResponse를 반환하는 메서드이니 반복해서 Line을 표시할 필요는 없겠네요
2. 파라미터명에 id가 있으니 메서드명에서는 제거해도 의미가 전달되겠네요